docs/writing-stories/args.md

```diff
@@ -10,7 +10,7 @@ Learn how and why to write stories in [the introduction](./introduction.md#using
 
```

Comment:
Perhaps we should add similar docs for `argTypes` to https://storybook.js.org/docs/react/api/argtypes?

Reply:
@kylegach sounds reasonable to me!

Reply:
Yeah, I got a bit hung up on the fact that we don't actually document story-level `argTypes` there (I guess `argTypes` at the story level generally speaking don't make a heap of sense, although it can be done). But I'll add something.